### PR TITLE
Added hono-geo-middleware

### PR DESCRIPTION
**Pull request by ktKongTong**
Listed hono-geo-middleware - https://github.com/ktkongtong/hono-geo-middleware in the 3rd party middleware section

### CODE REVIEW

File: docs/middleware/third-party.md [hunk 1]
```diff
@@ -43,3 +43,4 @@ Most of this middleware leverages external libraries.
 - [RONIN (Database)](https://github.com/ronin-co/hono-client)
 - [Sentry](https://github.com/honojs/middleware/tree/main/packages/sentry)
 - [tRPC Server](https://github.com/honojs/middleware/tree/main/packages/trpc-server)
+- [Geo](https://github.com/ktkongtong/tree/main/packages/hono-geo-middleware)
```
Comment (ktKongTong) [inline docs/middleware/third-party.md:46]:
```suggestion
- [Geo](https://github.com/ktkongtong/tree/main/packages/middleware)
```

File: docs/middleware/third-party.md [hunk 2]
```diff
@@ -43,3 +43,4 @@ Most of this middleware leverages external libraries.
 - [RONIN (Database)](https://github.com/ronin-co/hono-client)
 - [Sentry](https://github.com/honojs/middleware/tree/main/packages/sentry)
 - [tRPC Server](https://github.com/honojs/middleware/tree/main/packages/trpc-server)
+- [Geo](https://github.com/ktkongtong/tree/main/packages/middleware)
```
Comment (ktKongTong) [inline docs/middleware/third-party.md:46]:
```suggestion
- [Geo](https://github.com/ktkongtong/hono-geo-middleware/tree/main/packages/middleware)
```

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@ktKongTong That's interesting! Thanks.

**Pull request merged**